pkg/controller/controller.go

```diff
@@ -72,7 +71,7 @@ func NewManager(cfg *rest.Config, options ctrl.Options, kcp bool) (ctrl.Manager,
 	c := rest.CopyConfig(cfg)
 	cacheOptions := cache.Options{
 		SelectorsByObject: cache.SelectorsByObject{
-			&pipelinev1beta1.PipelineRun{}: {Label: labels.SelectorFromSet(map[string]string{artifactbuild.PipelineRunLabel: ""})},
+			&pipelinev1beta1.PipelineRun{}: {},
```

Comment:
fwiw, per our discussion my Monday evening / your Tuesday AM @stuartwdouglas, even with the removal of this label, the fact that we are using selectors by object at all means I had to go down the path of solving "atypical" integration between controller runtime and kcp, so that pain/effort was no wasted :-)

Reply:
This is only because we need some way of looking at the 'initial' build that kicks off the whole process. I need to look at its results so I can use them to create the ArtifactRequest objects.

Reply:
I can do it here

Reply:
cool thanks